# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515521364 28800
#      Tue Jan 09 10:09:24 2018 -0800
# Node ID bcb8ce5c4e164cf68b08b9d0cf593b84f7df3aad
# Parent  4db06f3bc34f3800c418aefc91b4bacb6eb9e2cb
annotation.m: Remove unused variable from update_arrow().

* annotation.m (update_arrow): Remove unused variable recursive.

diff --git a/scripts/plot/appearance/annotation.m b/scripts/plot/appearance/annotation.m
--- a/scripts/plot/appearance/annotation.m
+++ b/scripts/plot/appearance/annotation.m
@@ -1018,17 +1018,16 @@ function [x, y] = arrowcoordinates (h, n
   XY = pos(1:2).' .+ XY;
 
   x = XY(1,:).';
   y = XY(2,:).';
 
 endfunction
 
 function update_arrow (h, dummy, prop, hpa = [])
-  persistent recursive = false;
 
   nar = [];
   for ii = 1:numel (hpa)
     if (numel (hpa) == 2)
       nar = ii;
     endif
     if (hpa(ii))
       switch (prop)
